DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_Pagination.cshtml

```diff
@@ -1,33 +1,33 @@
-﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks
-@model BaseFrameworksPageViewModel
-
-<nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
-  <ul class="nhsuk-list nhsuk-pagination__list">
-    <div class="pagination-button-container">
-      <li class="nhsuk-pagination-item--previous" @(Model.Page == 1 ? "hidden" : "")>
-        <button class="nhsuk-pagination__link nhsuk-pagination__link--prev" type="submit" asp-action="@ViewContext.RouteData.Values["action"].ToString()" asp-route-page="@(Model.Page - 1)">
-          <svg class="nhsuk-icon nhsuk-icon__arrow-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
-            <path d="M4.1 12.3l2.7 3c.2.2.5.2.7 0 .1-.1.1-.2.1-.3v-2h11c.6 0 1-.4 1-1s-.4-1-1-1h-11V9c0-.2-.1-.4-.3-.5h-.2c-.1 0-.3.1-.4.2l-2.7 3c0 .2 0 .4.1.6z"></path>
-          </svg>
-          <span class="nhsuk-pagination__title">Previous</span>
-        </button>
-      </li>
-    </div>
-    <div class="page-indicator-container">
-      <span aria-live="polite" aria-atomic="true">
-        <span class="nhsuk-u-visually-hidden">page </span>
-        <span id="page-indicator">@Model.Page&nbsp;of&nbsp;@Model.TotalPages</span>
-      </span>
-    </div>
-    <div class="pagination-button-container">
-      <li class="nhsuk-pagination-item--next" @(Model.Page < Model.TotalPages ? "" : "hidden")>
-        <button class="nhsuk-pagination__link nhsuk-pagination__link--next" type="submit" asp-action="@ViewContext.RouteData.Values["action"].ToString()" asp-route-page="@(Model.Page + 1)">
-          <span class="nhsuk-pagination__title">Next</span>
-          <svg class="nhsuk-icon nhsuk-icon__arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
-            <path d="M19.6 11.66l-2.73-3A.51.51 0 0 0 16 9v2H5a1 1 0 0 0 0 2h11v2a.5.5 0 0 0 .32.46.39.39 0 0 0 .18 0 .52.52 0 0 0 .37-.16l2.73-3a.5.5 0 0 0 0-.64z"></path>
-          </svg>
-        </button>
-      </li>
-    </div>
-  </ul>
-</nav>
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model BaseSearchablePageViewModel
+
+<nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
+  <ul class="nhsuk-list nhsuk-pagination__list">
+    <div class="pagination-button-container">

```

Comment:
I think this has been done because the .nhsuk-pagination-item--next/previous elements have width 50% and we need ~33%, but yes I agree the html isn't correct.